packages/fast-usdc/test/snapshots/fast-usdc.contract.test.ts.md

```diff
@@ -1501,7 +1501,7 @@ Generated by [AVA](https://avajs.dev).
         'Operator Kit_kindHandle': 'Alleged: kind',
         PendingSettleTxs: {},
         PoolAccount: 'Vow',
-        SeenTxs: [],
+        SeenTxs: {},
```

Comment:
Non-trivial contents in the map would make me feel better about test coverage.
Please consider moving the snapshot until after the contract has seen some transactions.

Not that important.

Reply:
Done